<ui version="4.0" >
 <class>dEditJournal</class>
 <widget class="Q3MainWindow" name="dEditJournal" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Journal</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>328</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget18" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel3" >
            <property name="text" >
             <string>Name:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eName" />
          </item>
         </layout>
        </item>
        <item row="3" column="0" >
         <widget class="Q3TextEdit" name="eDescription" />
        </item>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="lType" >
            <property name="text" >
             <string>Type:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbType" >
            <item>
             <property name="text" >
              <string>Common</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Special</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Othes</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="textLabel4" >
          <property name="text" >
           <string>Description:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Aliases</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tAliases" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Documents</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QHBoxLayout" />
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="bAddDoc" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bRemoveDoc" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bMoveUp" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bMoveDown" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset/>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>56</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Access</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tRoles" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>2</number>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
          <property name="selectionMode" >
           <enum>Q3Table::Multi</enum>
          </property>
          <column>
           <property name="text" >
            <string>Read</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Write</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="Action" >
   <property name="iconText" >
    <string>Action</string>
   </property>
   <property name="name" stdset="0" >
    <string>Action</string>
   </property>
  </action>
  <action name="Action_2" >
   <property name="iconText" >
    <string>Action_2</string>
   </property>
   <property name="name" stdset="0" >
    <string>Action_2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mdtree.h</include>
  <include location="local" >atreeitems.h</include>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >mainform.h</include>
  <include location="local" >roleeditor.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>cbType</sender>
   <signal>activated(int)</signal>
   <receiver>dEditJournal</receiver>
   <slot>typeChange()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAddDoc</sender>
   <signal>clicked()</signal>
   <receiver>dEditJournal</receiver>
   <slot>addDoc()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bRemoveDoc</sender>
   <signal>clicked()</signal>
   <receiver>dEditJournal</receiver>
   <slot>removeDoc()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bMoveUp</sender>
   <signal>clicked()</signal>
   <receiver>dEditJournal</receiver>
   <slot>moveUp()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bMoveDown</sender>
   <signal>clicked()</signal>
   <receiver>dEditJournal</receiver>
   <slot>moveDown()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
